### ISSUE TITLE
Feat request: absolute path for auto import

### ISSUE THREAD

**Issue by nereuseng** [vuejs/vetur#1177]
- [x] I have searched through existing issues
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: Window 10 1809
- Vetur version: 0.18.0
- VS Code version: 1.32.3

## Problem

Currently the auto import is imported in relative path, is that any possible to add support auto import in absolute path or with alias? 
Example:
1. absolute path:
``` javascript
import { getStoreUserData } from './store/getter.types.js'
```
2. with alias
``` javascript
import { getStoreUserData } from '@/store/getter.types.js'
```

Besides, I find that another extension [auto import](https://marketplace.visualstudio.com/items?itemName=NuclleaR.vscode-extension-auto-import) was working in previous version, but It was broken after new version installed. Is that any possible to disable?

Thank you so much for contributing such a great work, I hope It will getting better!

**Comment by octref** [comment]
If you add path mapping, does it work? https://github.com/vuejs/vetur/blob/master/docs/FAQ.md#vetur-cant-recognize-components-imported-using-webpacks-alias

**Comment by nereuseng** [comment]
Nope. Alias work on importing .vue file, but not working on auto import.

![image](https://user-images.githubusercontent.com/11313394/55527558-24aac800-56cc-11e9-8e69-393c53ae76a5.png)

**Comment by praveenpuglia** [comment]
<details>
<summary> <strong>tsconfig.json</strong> </summary>
<pre><code>
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "strict": true,
    "jsx": "preserve",
    "importHelpers": true,
    "moduleResolution": "node",
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "resolveJsonModule": true,
    "esModuleInterop": true,
    "sourceMap": true,
    "baseUrl": ".",
    "types": ["webpack-env", "jest"],
    "paths": {
      "@/*": ["src/*"]
    },
    "lib": ["esnext", "dom", "dom.iterable", "scripthost"]
  },
  "include": [
    "src/**/*.ts",
    "src/**/*.tsx",
    "src/**/*.vue",
    "tests/**/*.ts",
    "tests/**/*.tsx"
  ],
  "exclude": ["node_modules"]
}</pre></code>
</details>

### Extension Version
```
Name: Vetur
Id: octref.vetur
Description: Vue tooling for VS Code
Version: 0.22.3
Publisher: Pine Wu
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=octref.vetur
```

### What's Happening
With the above **tsconfig.json** here's what I get.

- Aliases work when the import path is typed manually. So I write `@/` and then hit <kbd>Ctrl</kbd> + <kbd>Space</kbd> and I get the autocompletion as expected.

- If somewher in the code, say a method, I try to access anything that's exported in some other file and TS knows about it, it allows me to automatically import it. BUT, when it does that, it doesn't use absolute path starting from say `@/`. 

- In a TS file however, if I try to import anything that's exported in a file that doesn't exist in the same directory, VSCode does full path completion. Here's a screenshot.
<img width="745" alt="image" src="https://user-images.githubusercontent.com/313363/65064221-a1d9a900-d99d-11e9-8491-59b9205366b9.png">

- In a TS file itself however, if I import something that exists in file that resides in same directory, VSCode uses relative path like...
```
import something from './something.js';
```

### What do I expect?
I would love the autocompletion inside the .vue file for a TS/JS project to always use absolute path, with the root alias. 

### Why?
With relative paths, it becomes a pain to figure out at what level a thing is and then moving those file while refactoring becomes an even bigger pain because VSCode's automatic file path updates don't work for .vue files.

With an absolute path, it's absolutely easy to move around things.

**Comment by IWANABETHATGUY** [comment]
maybe you should try this vscode-extension https://marketplace.visualstudio.com/items?itemName=IWANABETHATGUY.path-alias

**Comment by octref** [comment]
Fixed by #1753.
